app/routes/index.js

```diff
@@ -11,15 +11,15 @@ export default Route.extend({
         page: { refreshModel: true },
     },
 
-    model(params) {
+    model() {
         if (!this.get('session.isAuthenticated')) {
             return this._emptyModels();
         }
         return hash({
             providers: this.get('store').query('preprint-provider', {
                 'filter[permissions]': 'view_actions,set_up_moderation',
             }),
-            actionsList: this.get('currentUser.user').then(this._getActions.bind(this, params.page)),
+            user: this.get('currentUser.user'),
```

Comment:
Instead of getting the user here, why not get `currentUser.user` in the component? Then the model could be just a list of providers instead of a hash, which would be nice.